### PR TITLE
Upload video

### PR DESCRIPTION
**Pull request by PechBunsinh**
- Create Upload Form use HTML and CSS

### CODE REVIEW

File: pj_movie/app/assets/javascripts/admin/customize.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
+window.onload = function () {
```
Comment (N310) [inline pj_movie/app/assets/javascripts/admin/customize.js]: Do you need this file?

File: pj_movie/app/assets/javascripts/admin/script.js [hunk 2]
```diff
@@ -0,0 +1,899 @@
+(function(){function qa(h,p){h.prototype=eb(p.prototype);h.prototype.constructor=h;h.base=p.prototype}function eb(h){function p(){}p.prototype=h;return new p}function Ya(h,p,D){"millisecond"===D?h.setMilliseconds(h.getMilliseconds()+1*p):"second"===D?h.setSeconds(h.getSeconds()+1*p):"minute"===D?h.setMinutes(h.getMinutes()+1*p):"hour"===D?h.setHours(h.getHours()+1*p):"day"===D?h.setDate(h.getDate()+1*p):"week"===D?h.setDate(h.getDate()+7*p):"month"===D?h.setMonth(h.getMonth()+1*p):"year"===D&&h.setFullYear(h.getFullYear()+
```
Comment (N310) [inline pj_movie/app/assets/javascripts/admin/script.js]: Do you need this file?

File: pj_movie/app/assets/stylesheets/admin/top_page.scss [hunk 3]
```diff
@@ -171,6 +167,7 @@
   margin-left: 246px;
   justify-content: center;
   align-items: center;
+  margin-left: 246px;
```
Comment (N310) [inline pj_movie/app/assets/stylesheets/admin/top_page.scss]: Do you need this change?

File: pj_movie/app/assets/javascripts/admin/customize.js (hunk 1 above)
Comment (PechBunsinh) [reply pj_movie/app/assets/javascripts/admin/customize.js]: no, I don't but I need this file for Purchase Detail branch

File: pj_movie/app/assets/javascripts/admin/script.js (hunk 2 above)
Comment (PechBunsinh) [reply pj_movie/app/assets/javascripts/admin/script.js]: no, I don't but I need this file for Purchase Detail branch

File: pj_movie/app/assets/stylesheets/admin/top_page.scss (hunk 3 above)
Comment (PechBunsinh) [reply pj_movie/app/assets/stylesheets/admin/top_page.scss]: no, I don't but I'll remove this

File: pj_movie/app/assets/stylesheets/admin/movies.scss [hunk 4]
```diff
@@ -0,0 +1,178 @@
+.upload_form{
```
Comment (N310) [inline pj_movie/app/assets/stylesheets/admin/movies.scss]: You should add a space after selector.
`.upload_form{` → `.upload_form {`

**Comment by N310** [comment]
One video have many tags.
So you have to change tag area.

File: pj_movie/app/assets/stylesheets/admin/movies.scss [hunk 5]
```diff
@@ -0,0 +1,178 @@
...
+      font-size: 24px;
+    }
+    .upload_content{
+      border-top: 1px solid #f3f3f3;
+      padding: 10px 10px 0px;
+      display: flex;
+      justify-content: flex-start;
+      flex-direction: column;
+      align-items: flex-start;
+      .video_info{
```
Comment (loemhuolay) [inline pj_movie/app/assets/stylesheets/admin/movies.scss]: Please add one more space before "{"

File: pj_movie/app/views/admin/movies/new.html.erb [hunk 6]
```diff
@@ -0,0 +1,46 @@
+<div class="upload_form">
+  <h1>Upload Video</h1>
```
Comment (loemhuolay) [inline pj_movie/app/views/admin/movies/new.html.erb]: In our rule we use Title 24px, but this you use h1, it mean 32px, please check again

File: pj_movie/app/assets/stylesheets/admin/movies.scss [hunk 7]
```diff
@@ -0,0 +1,178 @@
+.upload_form {
+  max-width: 100%;
+  height: auto;
+  display: flex;
+  justify-content: flex-start;
+  flex-direction: column;
+  h1{
```
Comment (loemhuolay) [inline pj_movie/app/assets/stylesheets/admin/movies.scss]: you should add space 
Ex: "h1{" to "h1 {"

File: pj_movie/app/assets/stylesheets/admin/movies.scss [hunk 8]
```diff
@@ -0,0 +1,178 @@
...
+          font-size: 14px;
+          width: 200px;
+          height: 40px;
+        }
+      }
+    }
+
+  }
+}
+.select_file{
```
Comment (loemhuolay) [inline pj_movie/app/assets/stylesheets/admin/movies.scss]: you should check ".select_file{"

File: pj_movie/app/assets/stylesheets/admin/movies.scss [hunk 9]
```diff
@@ -0,0 +1,178 @@
...
+}
+//==============width 1000px ===================
+@media only screen and (max-width:1000px) {
+  .upload_form {
+    max-width: 100%;
+    height: auto;
+    display: flex;
+    justify-content: flex-start;
+    flex-direction: column;
+    h1{
```
Comment (loemhuolay) [inline pj_movie/app/assets/stylesheets/admin/movies.scss]: h1

File: pj_movie/app/assets/stylesheets/admin/movies.scss [hunk 10]
```diff
@@ -0,0 +1,178 @@
...
+      font-size: 24px;
+    }
+    .upload_content {
+      border-top: 1px solid #f3f3f3;
+      padding: 10px 10px 0px;
+      display: flex;
+      justify-content: flex-start;
+      flex-direction: column;
+      align-items: flex-start;
+      .video_info{
```
Comment (loemhuolay) [inline pj_movie/app/assets/stylesheets/admin/movies.scss]: .video_info{

File: pj_movie/app/assets/javascripts/admin/jquery.tagsinput.js [hunk 11]
```diff
@@ -0,0 +1,343 @@
+(function($) {
```
Comment (N310) [inline pj_movie/app/assets/javascripts/admin/jquery.tagsinput.js]: Basically you shouldn't change import file.
If you want to change, you should use option of this plugin.

File: pj_movie/app/views/admin/movies/new.html.erb [hunk 12]
```diff
@@ -0,0 +1,71 @@
...
+      width: 'auto',
+    });
+
+// Uncomment this line to see the callback functions in action
+//			$('input.tags').tagsInput({onAddTag:onAddTag,onRemoveTag:onRemoveTag,onChange: onChangeTag});
+
+// Uncomment this line to see an input with no interface for adding new tags.
+//			$('input.tags').tagsInput({interactive:false});
+  });
+</script>
```
Comment (N310) [inline pj_movie/app/views/admin/movies/new.html.erb]: You should create new JS file and move this lines to the file.

File: pj_movie/app/assets/javascripts/admin/tagsInput.js [hunk 13]
```diff
@@ -0,0 +1,18 @@
...
+  alert("Added a tag: " + tag);
+}
+
+function onRemoveTag(tag) {
+  alert("Removed a tag: " + tag);
+}
+
+function onChangeTag(input,tag) {
+}
+
```
Comment (N310) [inline pj_movie/app/assets/javascripts/admin/tagsInput.js]: I think you don't use this lines.
Do you need?

File: pj_movie/app/controllers/admin/movies_controller.rb [hunk 14]
```diff
@@ -1,4 +1,6 @@
 class Admin::MoviesController < ApplicationController
+  def new
+  end
```
Comment (N310) [inline pj_movie/app/controllers/admin/movies_controller.rb]: You should add empty line.

File: pj_movie/app/views/admin/movies/new.html.erb [hunk 15]
```diff
@@ -0,0 +1,46 @@
...
+          </div>
+          <div class="input_textbox">
+            <div class="text_box">
+              <%= f.label :price, 'Price'%>
+              <%= f.text_field :name, class: 'form-control', :required => true %>
+            </div>
+          </div>
+          <div class="input_text">
+            <%= f.label :tag, 'Tags'%>
+			      <input id='tags' type='text' class='tags'></p>
```
Comment (N310) [inline pj_movie/app/views/admin/movies/new.html.erb]: You shouldn't use Tab, have to use space.

File: pj_movie/app/assets/stylesheets/admin/movies.scss [hunk 16]
```diff
@@ -1,3 +1,180 @@
...
+  .select_file {
+    input[type=text] {
+      width: 82.5%;
+      padding: 10px;
+      display: inline-block;
+      border: 1px solid #ccc;
+      border-radius: 4px;
+      box-sizing: border-box;
+    }
+  }
```
Comment (N310) [inline pj_movie/app/assets/stylesheets/admin/movies.scss]: Can you show `/admin/movies/new` page? This page have an error.
You have to add `}`.

**Pull request merged**